internal/disk/disk.go

```diff
@@ -53,19 +50,6 @@ const (
 	XBootLDRPartitionGUID = "BC13C2FF-59E6-4262-A352-B275FD6F7172"
 )
 
-var MountpointPolicies = pathpolicy.NewPathPolicies(map[string]pathpolicy.PathPolicy{
-	"/":     {Exact: true},
-	"/boot": {Exact: true},
-	"/var":  {},
-	"/opt":  {},
-	"/srv":  {},
-	"/usr":  {},
-	"/app":  {},
-	"/data": {},
-	"/home": {},
-	"/tmp":  {},
-})
```

Comment:
Can this change (moving the `MountpointPolicies`) be merged with the commit when the package split happened?  Not important though.

Reply:
Probably yes, I just did the split really in steps to not mess things up too much. Thus moving things around in multiple commits.

Reply:
No worries then.